hax/hax/handler.py

```diff
@@ -103,10 +121,22 @@ def pull_msg():
                         decorated(item)
                     elif isinstance(item, BroadcastHAStates):
                         LOG.info('HA states: %s', item.states)
-                        result: List[MessageId] = motr.broadcast_ha_states(
-                            item.states)
-                        if item.reply_to:
-                            item.reply_to.put(result)
+                        if item.is_broadcast_local:
+                            result: List[MessageId] = motr.broadcast_ha_states(
+                                item.states)
+                            if item.reply_to:
+                                item.reply_to.put(result)
+                        else:
+                            states_json = []
```

Comment:
Please consider refactoring if and else branches into separate functions.

Reply:
Okay.